frontend/src/components/ProgramForm/types.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ *
+ */
+
+/**
+ *
+ */
+export type ProgramData = {
+  name: string;
+  abbreviation: string;
+  type: string;
+  days: string[];
+  start: string;
+  end: string;
+  color: string;
+  renewal: string;
+  hourly: string;
+  sessions: string[][];
+};
+
+export type programColor = "teal" | "green" | "red" | "yellow" | "blue" | "violet" | "fuchsia";
```

Comment:
Just want to point out that we're storing the program color as hex values on backend. This is fine for now but will likely need modification when connecting with backend.

Reply:
because of the way tailwind works, you need to specify classvalues explicitly and cant define them dynamically, so i think this way works best and is most self-documenting. i plan to just map the colors to hex values when sending forms